src/main/webapp/app/entities/flair-bi/flair-bi.controller.js

```diff
@@ -1381,6 +1383,18 @@
             return editMode;
         }
 
+        function ngIfSettingsTable(v) {
```

Comment:
no need to rename this function as its not only meant for table

Reply:
I have not renamed the function
I have created a new function and its only use for table